src/createRollupFetchTransactionHash.ts

```diff
@@ -59,7 +59,7 @@ export async function createRollupFetchTransactionHash({
   const chainId = validateParentChain(publicClient);
 
   const fromBlock =
-    chainId in Object.keys(earliestRollupCreatorDeploymentBlockNumber)
+    chainId in earliestRollupCreatorDeploymentBlockNumber
```

Comment:
Although we are now searching from the block where the RollupCreator was deployed, it's still too many blocks to search through for some free tier L1 nodes. We might want to find a way to adjust the search limits and perform multiple searches.